src/panels/flow-histogram-react/FlowHistogramHelpers.ts

```diff
@@ -1,96 +1,217 @@
-import { SelectableValue } from "@grafana/data";
-import { FlowHistogramOptionsProps } from "./FlowHistogramTypes";
-
-const formatFlowHistogram = (histogramUnit: SelectableValue<string>, v) => {
-    let divisor = 1;
-
-    if (histogramUnit.label === 'KB') {
-        divisor = 1000
-    } else if (histogramUnit.label === 'MB') {
-        divisor = 1000000
-    } else if (histogramUnit.label === 'GB') {
-        divisor = 1000000000;
-    }
+/* eslint-disable no-restricted-imports */
+import { FlowHistogramOptionsProps, FlowPanelDataProcessed, FlowPanelUnitInfo } from "./FlowHistogramTypes";
+import _ from 'lodash';
+import moment from 'moment';
+import { DataPosition, FLowDataDirection, UnitInfo } from "./FlowHistogramContants";
 
-    return v / divisor
-}
-export const getFlowHistogramPlotData = (data, options: { flowHistogramOptions: FlowHistogramOptionsProps }) => {
-
-    let totalIn = 0;
-    let totalOut = 0;
-    let countedIn = 0;
-    let countedOut = 0;
-    data?.series?.forEach((sd) => {
-        let counterType;
-        if (sd.name.includes('(In)')) {
-            counterType = 'in'
-        } else if (sd.name.includes('(Out)')) {
-            counterType = 'out'
-        }
-        if (counterType) {
-            const valueField = sd.fields.find((f) => f.name === 'Value')
-            if (valueField) {
-                const valueArray = valueField.values.toArray()
-                valueArray.forEach((v) => {
-                    // OPTIONS LINK: Units. Convert Bytes to selected option.
-                    let formatted = formatFlowHistogram(options.flowHistogramOptions.units, v);
-                    if (counterType === 'in') {
-                        countedIn += 1;
-                        totalIn += formatted
-                    } else if (counterType === 'out') {
-                        totalOut += formatted
-                        countedOut += 1;
-                    }
-                })
+export const getFlowHistogramPlotData = (processedData: FlowPanelDataProcessed, options: { flowHistogramOptions: FlowHistogramOptionsProps }): any[] => {
```

Comment:
Nit: Try to avoid using a forced `any` return value. A Typescript error/warning with your return value is usually indicative of something you should fix versus override.